### PR TITLE
allow user to interrupt Python REPL

### PR DESCRIPTION
**Pull request by kevinushey**
### Intent

This PR ensures that the Stop button is displayed in the Console Pane when the Python REPL is busy, and clicking that will interrupt the associated Python computation.

Addresses https://github.com/rstudio/rstudio/issues/8763.
Addresses https://github.com/rstudio/rstudio/issues/8785.

### Approach

Listen for reticulate "busy" events, and use those to control the display + enabled-ness of the R interrupt command (which is handled appropriately when `reticulate` is running).

### QA Notes

Note: requires the development version of `reticulate` for interrupt support. Install it with:

```
remotes::install_github("rstudio/reticulate")
```

Then, try running a Python script with the contents:

```
import time
time.sleep(30)
```

and check that the associated computation can be interrupted.

https://user-images.githubusercontent.com/1976582/105775097-19690200-5f1b-11eb-990d-3bd36b169f73.mov

### CODE REVIEW

File: src/gwt/src/org/rstudio/studio/client/events/ReticulateEvent.java [hunk 1]
```diff
@@ -70,8 +76,9 @@ protected void dispatch(Handler handler)
    public static final Type<Handler> TYPE = new Type<>();
 
    // synchronize with SessionReticulate.R
-   public static final String TYPE_PYTHON_INITIALIZED = "python_initialized";
+   public static final String TYPE_PYTHON_INITIALIZED = "pythAon_initialized";
```
Comment (gtritchie) [inline src/gwt/src/org/rstudio/studio/client/events/ReticulateEvent.java:79]: Is this change intended?

Comment (kevinushey) [reply src/gwt/src/org/rstudio/studio/client/events/ReticulateEvent.java:79]: It definitely is not -- thanks for catching that.
